package/react-native-slider.podspec

```diff
@@ -17,22 +17,5 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/callstack/react-native-slider.git", :tag => "v#{s.version}" }
   s.source_files = "ios/**/*.{h,m,mm}"
 
-  s.dependency 'React-Core'
-
-  # This guard prevent to install the dependencies when we run `pod install` in the old architecture.
-  if ENV['RCT_NEW_ARCH_ENABLED'] == '1' then
-    s.compiler_flags = folly_compiler_flags + " -DRCT_NEW_ARCH_ENABLED=1"
```

Comment:
Due to removing this line, `folly_compiler_flags` from line 5 should also be removed.